download/download-pages.rkt

```diff
@@ -118,7 +118,10 @@
           @div[id: "m1_mac_explain"
                style: note-style]{
                @div{@b{M1 Mac users:}}
-               @list{Try a 64-bit ARM build from one of the @pre:installers{snapshot sites}.}}]
+               @list{The Mac OS (Intel 64-bit) variant works on macOS Big Sur 11.1 or above.
+               If it crashes, consider upgrading your operating system.
+
+               Alternatively, newer versions of Racket support Apple Silicon directly.}}]
```

Comment:
Could we say "current" here? The problem is people trying to download 8.0, right?